agent/config/config.go

```diff
@@ -315,6 +315,8 @@ func environmentConfig() (Config, error) {
 		seelog.Debugf("Setting instance attribute %v: %v", attributeKey, attributeValue)
 	}
 
+	containerMetadataEnabled := utils.ParseBool(os.Getenv("ECS_ENABLE_CONTAINER_METADATA"), false)
```

Comment:
I think the feature should be disabled by default to minimize its intrusiveness, however minimal, to general users. If users have a particular need for the container metadata for their applications, I think this should be done deliberately. However, I understand that we want this feature to have minimal impact anyways so having it enabled by default also makes sense.